model_list_dir/fisheye.json

```diff
@@ -1,14 +1,17 @@
 {
-  "authors": "Lisa Colburn",
-  "background_text": "An interactive tool for exploring economic data from West Coast commercial fisheries. Data summarized comes from the Economic Data Collection Program and PacFIN.",
-  "maintainer_name": "Erin Steiner",
-  "maintainer_email": "erin.steiner@noaa.gov",
+  "authors": "Erin Steiner, Amanda Phillips",
+  "contributors": "Melanie Harsch, Lisa Pfeiffer, Marie Guldin, Melissa Krigbaum, Ashley Vizek",
+  "background_text": "The FISHeries Economics Explorer (FISHEyE) is an interactive tool for exploring data from the West Coast Groundfish Trawl Catch Share Program. FISHEyE provides metrics to track and evaluate the catch share program. The metrics are not intended to be interpreted as causal, directional, nor normative but, rather, to facilitate the examination of trends and changes. Data used in FISHEyE comes from the Economic Data Collection (EDC) Program and the Pacific Fisheries Information Network (PacFIN). As part of the catch share program, participants of the fishery are required to complete EDC survey forms as stated in regulation 50 CFS 660.114. Data collection began in 2009, two years prior to implementing the catch share program. All vessels that participate in the catch share program must report data for all fisheries they participate in, including non-catch share fisheries.",
+  "maintainer_name": "Amanda Phillips",
+  "maintainer_email": "amanda.phillips@noaa.gov",
   "tool_abbreviation": "fisheye",
   "tool_name": "Fisheries Economic Explorer",
+  "references": ["Harsch, M., L. Pfeiffer, E. Steiner, and M. Guldin. 2018. Economic Performance Metrics : An Overview of Metrics and the Use of Web Applications to Disseminate Outcomes in the U.S. West Coast Groundfish Trawl Catch Share Program. NOAA technical memorandum NMFS-NWFS 143.","Steiner, E., A. Vizek, M. Guldin, M. Krigbaum, and L. Pfeiffer. 2021. Evaluating the Economic Performance of the U.S. West Coast Groundfish Trawl Catch Share Program. NOAA Tech Memo NMFS-NWFSC 169."],
   "online_app_link": "https://connect.fisheries.noaa.gov/fisheye/fisheyelandingpage.html",
+  "documentation_link": "https://connect.fisheries.noaa.gov/fisheye_literature/Literature.html",
   "active_development": true,
   "noaa_internal": true,
   "uses_github_releases": false,
   "toolbox_drawers": ["Human Dimensions"],
-  "keywords": ["Web App", "Economics"]
+  "keywords": ["Web App", "Economics", "West Coast Groundfish Trawl Catch Share Program"]
```

Comment:
same comment about keywords here!

Reply:
Ah, you can just delete the ones I added. Was trying to be more descriptive but didn't realize there was a list!

Reply:
My fault for not telling you! Ok, I will remove them.